packages/shared/src/dto/workflows/workflow-commons-fields.ts

```diff
@@ -74,7 +88,17 @@ export type StepDto = {
   type: StepTypeEnum;
 };
 
-export type WorkflowCommonsFields = {
+/**
+ * Common fields used by framework workflows to store schemas and metadata
+ */
+export type FrameworkCommonsFields = {
```

Comment:
This goes against our goal, which is to unify the framework and dashboard workflows. Which are the attributes that prevent us from defining a single unified workflow?

Reply:
where was that goal defined? 
there is already a DTO for code first, we should not alter DTO's for our API's to support specifically code first, IMO  the sync should reuse as much as possible and add on top of it the unuiqe features of code first